packet/src/test/java/org/jclouds/packet/features/ProjectApiMockTest.java

```diff
@@ -33,17 +31,17 @@
    public void testListProjects() throws InterruptedException {
       server.enqueue(jsonResponse("/projects.json"));
 
-      List<Project> projects = api.projectApi().list();
+      Iterable<Project> projects = api.projectApi().list().concat();
 
-      assertEquals(size(projects), 1); 
+      assertEquals(size(projects), 1);
       assertEquals(server.getRequestCount(), 1);
       assertSent(server, "GET", "/projects");
    }
```

Comment:
Change the test to something like [this](https://github.com/jclouds/jclouds/blob/master/providers/digitalocean2/src/test/java/org/jclouds/digitalocean2/features/ActionApiMockTest.java#L37-L48), to also verify that pagination works.